app/views/users/index.html.erb

```diff
@@ -0,0 +1,13 @@
+<div class="container">
+  <% @users.each do |user| %>
+    <div class= "user_info">
+      <div class="img">
+        <%= image_tag user.photo, alt: "User Photo", class: "user-photo"%>
+      </div>
+      <div class="user">
+        <h2><%= link_to user.name, user_path(user) %></h2>
+        <p><strong>Number of posts: </strong><%= user.posts.count %></p>
+      </div>
+    </div>     
+  <% end %>
+</div>
```

Comment:
- [ ] Kindly ensure the width of the items stretches the width of the screen like in [the wireframe provided](https://github.com/microverseinc/curriculum-rails/raw/main/blog-app/images/blog_root_users.png).

  ![image](https://github.com/Obimbo07/tdn/assets/36542986/a5c17a6f-b483-4c64-8062-2f25419812c6)

- [ ] **[OPTIONAL]** It would be nice to make this a partial and then import it in the views files as you need instead of repeating code.

Reply:
Fixed.
![image](https://github.com/Obimbo07/tdn/assets/93598189/d39009c1-ad97-48df-865c-b0013512ad58)